packages/rlc-codegen/src/interfaces.ts

```diff
@@ -35,3 +35,54 @@ export interface OperationAnnotations {
   isLongRunning?: boolean;
   isPageable?: boolean;
 }
+export interface RLCModel {
+  libraryName: string;
+  srcPath: string;
+  paths: Paths;
+  schemas?: Schema[];
+}
+export interface File {
+  path: string,
+  content: string
+}
+
+export enum SchemaContext {
```

Comment:
I wonder if we should use `Model` instead of Schema. Schema is heavily used in autorest and I think Model is more generic term